prep-html-syntax/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML-Syntax</title>
+  <body>
+    <h1 style="color: blue;">My favorite Star Wars character is General Grievous!</h1>
+<img src="https://images3.memedroid.com/images/UPLOADED324/5f3602864c612.jpeg"
+alt="General Kenobi as internet explorer with 4 buffering lightsabers"
+width="500" height ="600">
+    <p style="color: red;">General Grievous wields four lightsabers instead of one like other characters.</p>
+  </body>
```

Comment:
Hi Moses! Same here, be sure to make the `<body>` a direct child of `<html>` and not `<head>`.